lib/Service/CardService.php

```diff
@@ -351,6 +354,29 @@ public function update($id, $title, $stackId, $type, $owner, $description = '',
 
 
 		$card = $this->cardMapper->update($card);
+
+		$stack = $this->stackMapper->find($card->getStackId());
+		$board = $this->boardService->find($stack->getBoardId());
+		$boardLabels = $board->getLabels();
+		if ($boardLabels == null) {
+			$boardLabels = [];
+		}
+		foreach($card->getLabels() as $cardLabel) {
```

Comment:
I think we should only do all this if the card was actually moved to a different board, can we check that in any way?